website/content/docs/architecture/catalog/v2.mdx

```diff
@@ -0,0 +1,74 @@
+---
+layout: docs
+page_title: v2 Catalog API
+description: Learn about version 2 of the Consul catalog, which uses GAMMA specified resources. Learn how the v2 catalog corresponds to the v1 catalog and Kubernetes resources..
+---
+
+# v2 Catalog API
+
+<Warning>
+The v2 catalog API is in a beta release for testing and development purposes. Do not use the v2 catalog or multi-port services in secure production environments.
+</Warning>
```

Comment:
Could we potentially move these v1 and v2 pages up to the API level? It's not just the catalog api that is changing, but rather all the APIs are going to have v2 analogues. And if someone navigates to the API docs, that may be where we want to make clear to the user that you can either use the v1 or v2 api for consul and that those have some different features.

Reply:
V2 has a HTTP api also, so I think in the long term we want to show at the API level that these are different. Also there are catalog, mesh, and auth APIs that are supported in v2 so I think having a toggle of v1 or v2 api where the v1 api has all of the pages of the existing API under it and v2 api has all of the pages of v2 API reference under it would be good. But totally ok if we can't do that for the 1.17 docs.